api/ByteLengthQueuingStrategy.json

```diff
@@ -99,7 +99,7 @@
           "spec_url": "https://streams.spec.whatwg.org/#ref-for-blqs-high-water-mark①",
           "support": {
             "chrome": {
-              "version_added": "78"
```

Comment:
This is from https://github.com/mdn/browser-compat-data/pull/7841, also based on the collector. https://mdn-bcd-collector.appspot.com/tests/api/ByteLengthQueuingStrategy doesn't finish running to me, so could there be something wrong with the current test?

Reply:
That's really weird, the tests are completing no problem on my end...  Are there any console errors?

Reply:
Actually yes:

> Uncaught (in promise) DOMException: Failed to register a ServiceWorker for scope ('https://mdn-bcd-collector.appspot.com/resources/') with script ('https://mdn-bcd-collector.appspot.com/resources/serviceworker.js'): Operation has been aborted

I don't know when this can happen. Have you seen it before?

Reply:
I've seen it in early collector days, but I'm surprised that it's still happening now.  Is the same error also occurring on the mdn-bcd-collector.gooborg.com domain?

The first recommendation I have is to clear all of the app data and refresh, maybe that'll help?

Reply:
Clearing site data surprisingly didn't work, and it happens on https://mdn-bcd-collector.gooborg.com/tests/api/ByteLengthQueuingStrategy too.

Chrome 107.0.5304.110 on macOS.

Reply:
Hmm...  Is it possible you have some flags turned on that may be interfering?  I'm running the exact same version and am having no problems at all even after clearing the app data.  The page is working perfectly in BrowserStack, SauceLabs and LambdaTest.

Reply:
I don't know, but I also don't want to hold this up any longer. If the test works for you and you get these results, I'll trust that!